src/systems/light_controller/LightController.hh

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_SYSTEMS_LIGHTCONTROLLER_HH_
+#define IGNITION_GAZEBO_SYSTEMS_LIGHTCONTROLLER_HH_
+
+#include <ignition/gazebo/System.hh>
+#include <memory>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  // Forward declaration
+  class LightControllerPrivate;
+
+  /// \brief Light controller with a reference to a single light.
```

Comment:
It's not clear to me why we need the light controller system. Isn't the single light use case covered by `UserCommands`?